src/Components/Ignitor/src/BlazorClient.cs

```diff
@@ -376,6 +379,14 @@ private void OnEndInvokeDotNet(string message)
             NextDotNetInteropCompletionReceived?.Completion?.TrySetResult(null);
         }
 
+        private void OnAttachComponent(int componentId, string domSelector)
+        {
+            var call = new CapturedAttachComponentCall(componentId, domSelector);
+            Operations?.AttachComponent.Enqueue(call);
+
+            NextAttachComponentReceived?.Completion?.TrySetResult(call);
```

Comment:
Update the error callback to cause this to explode. We want to make sure stuff can fail fast.